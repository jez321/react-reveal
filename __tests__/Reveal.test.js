/*
 * React Reveal Test Suite
 *
 * Copyright Â© Roman Nosov 2016, 2017
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE.txt file in the root directory of this source tree.
 */
import Reveal from '../Reveal';

import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import { configure, shallow } from 'enzyme';
import React from 'react';

configure({ adapter: new Adapter() });


describe('Reveal', () => {
  it('renders a initial view', () => {
    const content = shallow(
      <Reveal effect="123" >
        <div>Test test</div>
      </Reveal>
    );
    expect(content.html()).toMatchSnapshot();
  });
});
