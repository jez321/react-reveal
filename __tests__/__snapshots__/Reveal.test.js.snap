// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reveal renders a initial view 1`] = `"<div class="react-reveal">Test test</div>"`;
